library/js/prescription-autocomplete/prescription-autocomplete.js

```diff
@@ -0,0 +1,129 @@
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+
+
+$(document).ready(function () {
+    $("#sendScript").toggle();
+    $('#pharmacy-autocomplete').autocomplete({
+
+        source: function (request, response) {
+            // Fetch data
+            let autocompleteUrl = $("#pharmacyselect").data('autocomplete-url');
+            $.ajax({
+                url: autocompleteUrl+'?term='+request.term,
+                type: 'GET',
+                dataType: "json",
+                data: {
+                    search: request.term
+                },
+                success: function (data) {
+                    var resp = $.map(data,function(obj){
+                        //console.log(obj.id);
+                     var   phadd = obj.id + ' ' +obj.name + ' ' + obj.line1 + ' ' +obj.city;
+                        return phadd;
+                    });
+                    response(resp);
+                }
+            });
+        }
+    });
+
+});
+
+let p = document.getElementById('savePharmacy');
+p.addEventListener('click', savePharm);
+
+function savePharm(e) {
+    top.restoreSession();
+    let pharmId = document.getElementById('pharmacy-autocomplete').value;
+    let id = pharmId.split(" ");
+    let setPharm = "../../src/Rx/Weno/PatientPharmacyController.php";//$("#savePharmUrl").data('savePharm-url');
+    if (id[0] > 0) {
+        $.ajax({
+            url: setPharm+'?id='+id[0],
+            type: 'GET',
+            dataType: 'json',
+            success: function (result) {
+                $("#savePharmacy").toggle();
+                getReturnJson(result)
+            },
+            error: function(xhr, status, error){
+                console.log(xhr);
+                console.log(status);
+                console.log(error);
+                console.warn(xhr.responseText);
+            }
+        });
+        e.preventDefault();
+        $("#sendScript").toggle();
+        return false;
+    } else {
+        $("#savePharmacy").toggle();
+        e.preventDefault();
+        $("#sendScript").toggle();
+    }
+}
+
+let s = document.getElementById('sendScript');
+s.addEventListener('click', sendScripts);
+function sendScripts(e) {
+    top.restoreSession();
+    $('#success').html("<i class='fa fa-refresh fa-spin fa-3x fa-fw'></i>");
+    let scripts = $('#prescriptIds').data('ids');
+    let sendRx = "../../interface/weno/transmitRx.php";
+    let request = [];
+    let responses = [];
+    if (scripts) {
+        request.push(
+      $.ajax({
+          url: sendRx+'?scripts='+scripts,
+          type: 'GET',
+          success: function (response) {
+              response.setHeader("Set-Cookie", "HttpOnly;Secure;SameSite=None");
+              parser = new DOMParser();
+              responses.push(response);
+              var announce = "Send Complete - Prescription(s) Return Status + Sign Off";
```

Comment:
translate this. there is a javascript function xl always available:
```
var announce = xl("Send Complete - Prescription(s) Return Status + Sign Off");
```

Reply:
Actually disregard this. This is US centric and the xl function likely will break if try to port it back to 5.0.2 patch.

Reply:
Aye!